docs/concepts/object-metadata/statefulsets.md

```diff
@@ -0,0 +1,172 @@
+---
+---
+
+{% capture overview %}
+**Stateful Sets are a beta feature in 1.5. This feature replaces the deprecated 
+Pet Sets feature from 1.4. Users of Pet Sets are referred to the 1.5 
+[Upgrade Guide](/docs/tasks/stateful-set/upgrade-from-petsets-to-stateful-sets/)
+for further information on how to upgrade existing Pet Sets to Stateful Sets.**
+
+A Stateful Set is a Controller that ensures that, at most, a given number of
```

Comment:
IMO this overview is too long. Suggest make it very short (just one sentence or one paragraph), and move these to a section, such as "example" or "components of Stateful Set" or "Anatomy of Stateful Set".